src/index.ts

```diff
@@ -183,8 +189,32 @@ class QubitSDK {
             !(ignoreSegments == null),
             ignoreSegments || false
         )
-            .then(experiences => experiences.map(e => ({...e, shown: () => { NativeModules.QubitSDK.experienceShown(e.callback || '')} })))
+            .then(experiences => experiences.map(e => ({
+                ...e, 
+                shown: () => { NativeModules.QubitSDK.experienceShown(e.callback || '') } 
+            })))
+    }
+
+
+    public getPlacement(
+        placementId: string,
+        mode?: string,
+        attributes?: string,
+        campaignId?: string,
+        experienceId?: string
+    ) : Promise<Placement> {
+        return NativeModules.QubitSDK.getPlacement(
+            placementId,
+            mode,
+            attributes,
+            campaignId,
+            experienceId
+        )
+             .then(placement => ({
+                ...placement,
+                impression: () => { NativeModules.QubitSDK.placementImpression(placement.impression || '') },
```

Comment:
According to documentation of BridgeAPI, NativeModules.QubitSDK.getPlacement method returns placement as:
```
{
	  "data": {
	    "placementContent": {
	      "content": {
	        "image": "https://image.store.com/images/example.jpeg",
	        "message": "Hello World",
	        "url": "https://www.qubit.com"
	      },
	      "callbacks": {
	        "impression": "https://api.qubit.com/placements/callback?data=ggW4eyJtZXRhIjp7ImlkIjo",
	        "clickthrough": "https://api.qubit.com/placements/callback?data=mQW4eyJtZXRhIjp7Imlkx"
	      }
	    }
	  }
	}
```
If documentation is correct, url of impression callback should be:
`placement.data.placementContent.callbacks.impression` and not simple `placement.impression`.

Reply:
Updated that call following new placement structure:
```
  {
        "image": "https://image.store.com/images/example.jpeg",
        "message": "Hello World",
        "url": "https://www.qubit.com"
        "impressionUrl": "https://api.qubit.com/placements/callback?data=ggW4eyJtZXRhIjp7ImlkIjo",
        "clickthroughUrl": "https://api.qubit.com/placements/callback?data=mQW4eyJtZXRhIjp7Imlkx"
     }
```